src/dosbox.cpp

```diff
@@ -781,19 +781,22 @@ void DOSBOX_Init()
 	secprop->AddInitFunction(&DMA_Init);
 	secprop->AddInitFunction(&VGA_Init);
 	secprop->AddInitFunction(&KEYBOARD_Init);
+	secprop->AddInitFunction(&PCI_Init); // PCI bus
 
-	secprop=control->AddSection_prop("pci", &PCI_Init, false); // PCI bus
-
-	secprop->AddInitFunction(&VOODOO_Init, false);
+	secprop = control->AddSection_prop("voodoo", &VOODOO_Init, false);
 
 	const char* voodootypes[] = { "12mb", "4mb", "disabled", nullptr };
-	pstring = secprop->Add_string("voodoo", only_at_start, "12mb");
+	pstring = secprop->Add_string("voodoo_memsize", only_at_start, "12mb");
 	pstring->Set_values(voodootypes);
-	pstring->Set_help("RAM amount of emulated Vodooo 3dfx card.");
+	pstring->Set_help("Memory size of the emulated Vodooo 3dfx card ('12mb' by default).");
 
 	pint = secprop->Add_int("voodoo_perf", only_at_start, 0);
-	pint->SetMinMax(0, 4);
-	pint->Set_help("Toggle performance optimizations for Vodooo 3dfx emulation (0 = none, 1 = use multi-threading, 2 = disable bilinear filter, 3 = both).");
+	pint->SetMinMax(0, 3);
+	pint->Set_help("Performance optimizations avaiable when emulating the Vodooo 3dfx card:\n"
+	               "   0:  No optimizations (default).\n"
+	               "   1:  Multi-threading.\n"
+	               "   2:  Disable bilinear filtering.\n"
+	               "   3:  All optimizations (both 1 and 2).");
```

Comment:
So this is a software-only renderer, right? Absolutely zero OpenGL is involved? Should we make that clear somewhere in the descriptions that is is a pure software-based emulation of the Voodoo card that requires no OpenGL?

Reply:
Added:
```c++
pint->Set_help("Performance optimisations to use when emulating the 3dfx Voodoo card:\n"
               "Note: emulation is software-based and does not use host-level OpenGL calls.\n"
               "   0:  No optimizations.\n"
               "   1:  Multi-threading (default).\n"
               "   2:  Disable bilinear filtering.\n"
               "   3:  All optimizations (both 1 and 2).");
```

@kklobe : I've tentatively enabled threading as the default optimization; can't remember which we decided was best overall.